docs/userguide/running-unit-tests.md

```diff
@@ -376,25 +434,13 @@ Examples (based on above sample test suite)
 select * from table(ut.run(a_tags => '(api|fast)&!complex'));
 ```
 
-which is equivalent of legacy calling:
-
-```sql linenums="1"
-select * from table(ut.run(a_tags => 'api,fast,-complex'));
-```
-
 or 
 
 ```sql linenums="1"
-select * from table(ut.run(a_tags => '(api|fast)&(!complex&!test1)'));
-```
-
-which is equivalent of legacy calling:
-
-```sql linenums="1"
-select * from table(ut.run(a_tags => 'api,fast,-complex,-test1'));
+select * from table(ut.run(a_tags => '(api|fast)&!complex&!test1'));
```

Comment:
Is the negation supported on groups?
so that one could use `(api|test)&!(complex|test1)` instead of `(api|test)&!complex&!test1` ?

I think it is worth mentioning in documentation if they are supported or not.

Reply:
It is supported. All is happening it will add not before sql part.

Reply:
I will update documentation for this.